### DISCUSSION TITLE
Is Our Master Branch Protected?

### DISCUSSION THREAD

**Discussion by zicklag** [fishfight/punchy#169] [General]
@odecay do we have our master branch protected? i.e. no force pushes to master?

I'm just making sure because I was intentionally force pusing to master in my fork, to test some CI stuff, and realized how horrible it could be if I accidentally force pushed to the punchy master and it wasn't blocked.

**Comment by zicklag** [comment]
And as I say that I accidentally push a v0.3.0 tag to Punchy. :/

Sorry, my bad! I edited my local git refs so it shouldn't happen accidentally again. It happened because punchy was my `origin` so a push without a specified remote automatically goes to origin.

I found out how to delete the tag and I canceled the CI jobs so nothing happened. :grin:

**Reply by odecay** [reply]
wew crisis averted, yea sounds like a good reason to set up protected master though. (I have my repos set as my origin and fishfight/punchy as upstream)

**Reply by zicklag** [reply]
Yeah, again, so sorry about that. I switched punchy to `upstream` after that. :+1:

**Reply by odecay** [reply]
no worries all is well

**Comment by zicklag** [comment]
We may also want to add a protected tag rule for `v*` tags.

**Comment by 64kramsystem** [comment]
Protection makes sense to me. :smile:

**Comment by odecay** [comment]
master and v* protected
